Makefile

```diff
@@ -83,29 +83,29 @@ dbdocs:
 	$(DOCKER_CMD) dot -Tpdf doc/database/ledgersmb.dot -o doc/database/ledgersmb.pdf
 
 npm_install:
-	$(DOCKER_CMD) npm $(NPM_COMMAND) --no-save
+	$(DOCKER_CMD) cd UI && npm $(NPM_COMMAND) --no-save
 
 dojo: npm_install
-	$(DOCKER_CMD) npm run build
+	$(DOCKER_CMD) cd UI && npm run build
 
 devdojo: npm_install
-	$(DOCKER_CMD) npm run build:dev
+	$(DOCKER_CMD) cd UI && npm run build:dev
 
 js: npm_install
-	$(DOCKER_CMD) npm run build
+	$(DOCKER_CMD) cd UI && npm run build
 
 jsdev: npm_install
-	$(DOCKER_CMD) npm run build:dev
+	$(DOCKER_CMD) cd UI && npm run build:dev
```

Comment:
Doesn't this cause the "npm" command to be run outside of the docker environment?

Reply:
I don't get that. How can you be so sure? Do you have npm installed inside and outside the container? Or just inside?

Reply:
Fixed.